Packs/iDefense/Integrations/FeediDefense/FeediDefense.py

```diff
@@ -131,8 +132,10 @@ def main():
     params = {k: v for k, v in demisto.params().items() if v is not None}
 
     filters: Dict[str, Optional[Union[str, list]]] = build_feed_filters(params)
-    indicators_type: list = params.get('indicator_type', []) if len(params.get('indicator_type', [])) \
+    indicator_type_field = argToList(params.get('indicator_type', []))
+    indicators_type: list = indicator_type_field if len(indicator_type_field) \
```

Comment:
Since this parameter is mandatory, then `argToList` is sufficient. there's no way that this field will be empty.